revokefs/main.c

```diff
@@ -20,7 +20,7 @@
  * Boston, MA 02111-1307, USA.
  */
 
-#define FUSE_USE_VERSION 26
+#define FUSE_USE_VERSION 31
```

Comment:
Would it make sense to inject this macro through build system instead? Then you could use right value depending on libfuse and use this variable for compat code.

Reply:
#4326 does as suggested here.